lib/linguist/languages.yml

```diff
@@ -6184,7 +6184,19 @@ Scilab:
   - ".tst"
   tm_scope: source.scilab
   ace_mode: text
-  language_id: 344
+  language_id: 344  
+Scpel:
+  type: programming
+  color: "#650791"
+  extensions:
+  - ".co"
+  - ".ho"
+  - ".h"
+  tm_scope: source.co
```

Comment:
Don't guess. If this is to use the C grammar, this should be the scope from the C grammar.